### PR TITLE
Simplify the setup of new projects

### PR DESCRIPTION
**Pull request by corneliusboehm**
Remove the separate pages for creating or importing projects, because since #102 classes and tags can be added/edited on the project details page.

### Adding new projects
The projects overview home page has been extended to contain a form for adding new projects or importing existing ones:
image

If the given path exists, we check if there is an existing project config file and otherwise create a new one, same for the required sub-directories.

### Fixing outdated projects
Projects with an outdated path can now also be edited right on the projects overview page:
image

### Classes and tags explanation
An explanatory message box (previously on the new project page) is added to the project details page:
image

### CODE REVIEW

**Review by nahuakang — COMMENTED** [review]
One nit-pick below:

In the case where I had a project name (`AAA`) but its project path no longer exists (as shown on the left panel), I should be able to re-use the project name `AAA` with a new project path `/some/path/to/AAA` and create a new `AAA` project without having to first delete the `AAA` project?

Also the `Another project is already initialized in this location` conflicts with the `This path no longer exists` since that directory is gone already and no project is in this location :thinking: 

Screenshot from 2021-02-04 11-07-37

**Comment by corneliusboehm** [comment]
@nahuakang Thanks for the in-depth testing! In your case above I would suggest though that you either update the path on the left side (if the project has been moved) or first remove the old one and then create a new one (if you want to start from scratch).

**Review by nahuakang — APPROVED** [review]
In that case, this PR works and should be merged :)

File: tools/sense_studio/sense_studio.py [hunk 1]
```diff
@@ -147,106 +147,67 @@ def remove_project(name):
...
-def create_new_project():
+@app.route('/setup-project', methods=['POST'])
+def setup_project():
     """
     Add a new project to the config file. Can also be used for updating an existing project.
     """
     data = request.form
-    name = data['name']
+    name = data['projectName']
     path = data['path']
```
[the remark this answers is not in the document]
Comment (corneliusboehm) [reply tools/sense_studio/sense_studio.py:172]: These are form fields that are always there so far. Maybe we can change that if this method is used in more places.

**Pull request merged**